docs/commands.rst

```diff
@@ -127,13 +127,28 @@ Specifying 0 as the target version will revert all migrations.
 
         $ phinx rollback -e development -t 0
 
+To rollback all migrations to a specific date then use the ``--date``
+parameter or ``-d`` for short.
+
+.. code-block:: bash
+
+        $ phinx rollback -e development -d 20120103
```

Comment:
Extend this to show that you can (for example) rollback to 2016 by `-d 2016` or 3AM today by `-d 2016-07-19 03`.

Essentially, `Y[m[d[H[i[s]]]]]`.

I wonder about supporting `-d 3pm`, `-d yesterday 2pm`, `-d 2 hours ago`, etc.

And should format of the -d parameter be the same as -t